include/RAJA/pattern/params/reducer.hpp

```diff
@@ -66,6 +64,14 @@ struct limits<RAJA::expt::ValLoc<T>> {
 
 namespace RAJA
 {
+namespace detail
+{
+#if defined(RAJA_CUDA_ACTIVE)
+using device_mem_pool_t = RAJA::cuda::device_mempool_type;
+#elif defined(RAJA_HIP_ACTIVE)
+using device_mem_pool_t = RAJA::hip::device_mempool_type;
+#endif
+} //namespace detail
```

Comment:
@MrBurmark I moved it closer to where it is being used, let me know if there is a preference.

Reply:
If its only used here you could put it in the expt and detail namespaces right above Reducer.